keyboards/mechlovin/zed65/rgb/readme.md

```diff
@@ -0,0 +1,28 @@
+# zed65rgb
+
+![Zed65RGB](https://i.imgur.com/IbsSLb6h.jpeg)
```

Comment:
This image URL is same image URL of `keyboards/mechlovin/zed60/rev1/readme.md` - could URL of an image of the actual PCB in question be inserted here?